src/plugins/data_view_editor/public/components/data_view_flyout_content_container.tsx

```diff
@@ -32,6 +28,18 @@ const DataViewFlyoutContentContainer = ({
     services: { dataViews, notifications, http },
   } = useKibana<DataViewEditorContext>();
 
+  const dataViewEditorService = useRef(
```

Comment:
The intention is to have a stable reference to a single service for the life of the component.